### ISSUE TITLE
On Win7 RDP there is no window border

### ISSUE THREAD

**Issue by rstefko** [adospace/fluent-ui-xaml#9]
![image](https://user-images.githubusercontent.com/4289079/228220179-0fd89734-a7c3-49db-8955-d861a89c1347.png)

**Comment by adospace** [comment]
yes, I think the problem is with the shadow that's not sent through the RDP, I guess you can solve it in your app using a border.

**Comment by rstefko** [comment]
Yes, but I have to edit the template. See the pull request.
